src/main/java/org/takes/rs/RsWithStatus.java

```diff
@@ -107,15 +110,21 @@ public RsWithStatus(final Response res, final int code,
                 )
             );
         }
-        return new Joined<>(
-            new FormattedText(
-                "HTTP/1.1 %d %s", status, reason
-            ).asString(),
-            new Filtered<>(
-                item -> new Not(
-                    new StartsWith(item, "HTTP/")
-                ).value(),
-                origin.head()
+        return new Mapped<>(
+            Text::asString,
```

Comment:
@andreoss Why don't you use `Text::toString` ?

Reply:
@baudoliver7 `toString` is not `Text` method, basically it would be the same as `Object::toString`. `Mapped` handles checked exception here.

Reply:
@andreoss But, if `toString()` is implemented in Text, we can use it (I think).

Reply:
@baudoliver7 `Text` only has `asString`. https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/Text.java#L39.